app/assets/javascripts/discourse/app/templates/modal/edit-slow-mode.hbs

```diff
@@ -13,22 +13,12 @@
     }}
   </div>
 
-  {{#if showCustomSelect}}
-    <div class="control-group">
-      {{d-icon "hourglass-end"}}
-      {{input value=hours type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.hours")}}
-      {{input value=minutes type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.minutes")}}
-      {{input value=seconds type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.seconds")}}
-    </div>
-  {{/if}}
-
-  {{#if model.slow_mode_seconds}}
-    <div class="alert alert-info">
-      <b>
-        {{i18n "topic.slow_mode_update.current" hours=hours minutes=minutes seconds=seconds}}
-      </b>
-    </div>
-  {{/if}}
+  <div class="control-group">
+    {{d-icon "hourglass-end"}}
+    {{input value=hours type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.hours")}}
+    {{input value=minutes type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.minutes")}}
+    {{input value=seconds type="number" class="input-small" placeholder=(i18n "topic.slow_mode_update.seconds")}}
+  </div>
```

Comment:
Showing that input fields all the time feels weird to me. Why would I want to see or edit any of those fields when I just selected "1 week"? I'd prefer if it's only visible when the custom option is selected.

Reply:
My bad. I misunderstood your feedback.

Reply:
No worries. My request probably was too confusing.